pkgs/applications/science/electronics/xyce/default.nix

```diff
@@ -0,0 +1,169 @@
+{ stdenv
+, fetchFromGitHub
+, fetchgit
+, lib
+, autoconf
+, automake
+, bison
+, blas
+, flex
+, fftw
+, gfortran
+, lapack
+, libtool_2
+, mpi
+, suitesparse
+, trilinos # if compiling with MPI so trilinos must be! Pass the correct one.
```

Comment:
This should instead be an assertion. Put `withMPI` into the `passthru` attribute of `trilinos` and compare it like 

```nix
assert withMPI -> trilinos.withMPI;
```

Reply:
Good idea. Done. Thanks.